spec/integration/support/inspect_unmanaged_files_examples.rb

```diff
@@ -61,7 +62,8 @@ def parse_md5sums(output)
 Inspecting unmanaged-files...
  -> Found 0 files and trees...\r\033\[K -> Found 0 files and trees...\r\033\[K -> Extracted 0 unmanaged files and trees.
 EOF
-      expect(normalize_inspect_output(@machinery_output)).to include(expected)
+      expect(normalize_inspect_output(@machinery_output[0])).to include(expected)
+      expect(@machinery_output[1]).to include("contains invalid UTF-8 characters")
```

Comment:
Indexing the output is not very explicit. I would very much prefer something like `expect(@machinery_output.stdout).to include(expected_stdout)`.